amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/setup/README.md

```diff
@@ -0,0 +1,87 @@
+    
+
+## Step 2.  Run the precursor spin-up case
+
+The first part of the ABL simulation will allow the turbulence to initialize and the boundary layer profile to develop.  The input file for this initial simulation is given by [convective_abl.inp](../input_files/convective_abl.inp), and will last for 15,000 seconds.
+    
+To launch this case, navigate to the right diretory and create a submission script:
+    
+```bash
+$ cd BENCHMARKDIR/amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/input_files
+$ vi submit.sh
+```
+
+The submission script (called `submit.sh` above) will resemble something similar to the file below for slurm based HPC clusters.   Edit both the location of the `amr_wind` executable and any modules that might be required for the executable to run.
+    
+```bash
+#!/bin/bash
+#SBATCH --nodes=8
+#SBATCH --time=47:59:59        # Wall clock time (HH:MM:SS) - once the job exceeds this time, the job will be terminated (default is 5 minutes)
+#SBATCH --job-name=ABL # Name of job
+#SBATCH --partition=batch      # partition/queue name: short or batch
+#SBATCH --qos=normal           # Quality of Service: long, large, priority or normal 
+export nodes=$SLURM_JOB_NUM_NODES
+
+# Load any required modules here
+#module purge
+#module load ...
+
+# Edit the location of the amr_wind executable here:
+export EXE=/projects/wind_uq/lcheung/AMRWindBuilds/hfm.20250211/amr-wind/build/amr_wind
+
+# Number MPI processes to run on each node (a.k.a. PPN)
+export cores=112
+export ncpus=$((nodes * cores))
+export OMP_PROC_BIND=spread 
+export OMP_PLACES=threads
+
+time mpiexec --bind-to core --npernode $cores --n $ncpus $EXE convective_abl.inp 
+```
+
+Lastly, launch the simulation on the cluster with a command similar to
+    
+```bash
+$ sbatch submit.sh
+```
+
+## Step 3.  Launch the statistics and sampling run.
+
+After completing step 2, the ABL simulation is run for an additional 5,000 seconds to gather statistics and boundary inflow data.  The input file for this second run is given by [convective_abl_bndry.inp](../input_files/convective_abl_bndry.inp).  To launch this case, use a similar submission as above, but edit the last line to use the new input file:
+
+```bash
+time mpiexec --bind-to core --npernode $cores --n $ncpus $EXE convective_abl_bndry.inp 
+```
+
+
+## Optional: using the jupyter notebook
+
+If you're interested in modifying the AMR-Wind input file, for instance, to change the sampling plane locations, you can use the Jupyter notebook [convectiveABL_setup.ipynb](convectiveABL_setup.ipynb).  This will allow you to visualize the locations of the planes relative to the turbine position and wind direction, and will generate the AMR-Wind input file [convective_abl.inp](../input_files/convective_abl.inp) at the end of the process.
+
+![ABL domain side view](../results/images/convectiveABL_sideview.png)
```

Comment:
I'm unclear on the what the colors in the figure.  Do the legend entries go with colors?

Reply:
Oh yes, that was missing explanation.  I just added some comments to explain what the sampling planes and turbine locations that are shown in that figure.